Packs/LogRhythm/Integrations/integration-LogRhythm.yml

```diff
@@ -354,21 +382,45 @@ script:
     };
 
     var sendRequest = function(envelope, soapAction, apiUrl) {
+        username = params.Credentials.identifier.split('\\')[1]
```

Comment:
why do we need the `\` ?

Reply:
Because that's how we are expected to send it when using NTLM for some reason